### PR TITLE
FAB-5367 read updates

### PR DESCRIPTION
**Pull request by jgielstra-cs**

### CODE REVIEW

**Comment by jgielstra-cs** [comment]
( Seem a bit sparse.. )
Description
Installation/Requirements
Running

File: templates/job/docs/__skillname__/README.md [hunk 1]
```diff
@@ -4,6 +4,22 @@ This workspace template demonstrates a simple job skill skeleton that simply pri
 
 ## Building and Publishing the Skill
 
-In order to build the job skill Docker image, simply select "Build Skills" from the Cortex menu in the IDE.  You can also right-click on the `skill.yaml` file for the skill and select "Build Skill" from the Cortex context menu.
+### Using the cortex cli
+
+Building skill(s)
+
+```
+cortex workspaces build
```
Comment (jlara-cs) [inline templates/job/docs/__skillname__/README.md:14]: To build all docker images in the workspace:
```
cortex workspaces build
```

To build all docker images in the folder:
```
cortex workspaces build [folder]
```

To build only the docker image for the specified skill:
```
cortex workspaces build --skill <skill name>
```

File: templates/job/docs/__skillname__/README.md [hunk 2]
```diff
@@ -4,6 +4,22 @@ This workspace template demonstrates a simple job skill skeleton that simply pri
...
+```
+
+Publishing  skill and docker image
```
Comment (jlara-cs) [inline templates/job/docs/__skillname__/README.md]: extra space

File: templates/job/docs/__skillname__/README.md [hunk 3]
```diff
@@ -4,6 +4,22 @@ This workspace template demonstrates a simple job skill skeleton that simply pri
...
+
+```
+cortex workspaces publish
```
Comment (jlara-cs) [inline templates/job/docs/__skillname__/README.md:34]: To publish all docker images and skills in the workspace:
```
cortex workspaces publish
```

To publish all docker images and skills in the folder:
```
cortex workspaces publish [folder]
```

To publish only the specified skill and its docker image:
```
cortex workspaces publish --skill <skill name>
```

File: templates/job/docs/__skillname__/README.md [hunk 4]
```diff
@@ -4,6 +4,22 @@ This workspace template demonstrates a simple job skill skeleton that simply pri
...
+```
+
+### Using the VS code extension (CognitiveScale.cortex-code)
+
+In order to build the daemon skill Docker image, simply select "Build Skills" from the Cortex menu in the IDE.  You can also right-click on the `skill.yaml` file for the skill and select "Build Skill" from the Cortex context menu.
```
Comment (jlara-cs) [inline templates/job/docs/__skillname__/README.md]: daemon -> job

File: templates/online-prediction/docs/__skillname__/README.md [hunk 5]
```diff
@@ -7,6 +7,22 @@ Note:
 
 ## Building and Publishing the Skill
 
+### Using the cortex cli
```
Comment (jlara-cs) [inline templates/online-prediction/docs/__skillname__/README.md:10]: same comments as in templates/job/docs/__skillname__/README.md

File: templates/daemon/docs/__skillname__/README.md [hunk 6]
```diff
@@ -4,7 +4,22 @@ This workspace template demonstrates a simple daemon skill skeleton that simply
 
 ## Building and Publishing the Skill
 
+### Using the cortex cli
```
Comment (jlara-cs) [inline templates/daemon/docs/__skillname__/README.md:7]: same comments as in templates/job/docs/__skillname__/README.md. except for the daemon to job comment as this is a daemon

**Review by jlara-cs — APPROVED** [review]
lgtm. left some comments

**Pull request merged**